### PR TITLE
Modify dialog messages to display standard messages (Fixed #1290)

### LINKED ISSUE #1290: Modify dialog messages to show standard messages
**Actual Behaviour**

Currently, the messages shown in the dialog have grammatical errors.

**Expected Behaviour**

These messages should be grammatically correct and should be to the point to let the user know what message the dialog aims to convey.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by arundhati24**
Fixes #1290 

Changes: Updated dialog message strings to show standard messages and actions.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
     <!--General Strings-->
     <string name="action_about_us">About Us</string>
+    <string name="action_confirm_log_out">LOG OUT</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: No need for explicit caps

Comment (arundhati24) [reply app/src/main/res/values/strings.xml]: @iamareebjamal Removed the explicit caps. Please review.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -166,7 +167,7 @@
     <string name="unknown_answer">I don\'t know.</string>
 
     <!-- Login Activity-->
-    <string name="continue_sign_up">Continue Registration</string>
+    <string name="continue_sign_up">Continue</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: Can be removed, there already is continue

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -191,6 +192,8 @@
     <!-- Settings Activity-->
     <string name="Continue">Continue</string>
 
+    <string name="dialog_log_out">Are you sure you want to log out?</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: logout_confirmation

**Pull request merged**